src/main/resources/static/error/403.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>403</title>
+</head>
+<body>
+<h1>Congrats! You have problem now, because</h1>
+<h2>This page with 403 code</h2>
+<img src="http://i2.kym-cdn.com/photos/images/facebook/000/277/976/9e3.jpg">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)